packages/twenty-front/src/modules/object-record/record-filter/utils/compute-empty-record-gql-operation-filter/for-composite-field/computeEmptyGqlOperationFilterForEmails.ts

```diff
@@ -0,0 +1,92 @@
+import { FieldMetadataItem } from '@/object-metadata/types/FieldMetadataItem';
+import {
+  EmailsFilter,
+  RecordGqlOperationFilter,
+} from '@/object-record/graphql/types/RecordGqlOperationFilter';
+
+import { RecordFilter } from '@/object-record/record-filter/types/RecordFilter';
+import { isNonEmptyString } from '@sniptt/guards';
+
+export const computeEmptyGqlOperationFilterForEmails = ({
+  recordFilter,
+  correspondingFieldMetadataItem,
+}: {
+  recordFilter: RecordFilter;
+  correspondingFieldMetadataItem: Pick<FieldMetadataItem, 'name' | 'type'>;
+}): RecordGqlOperationFilter => {
+  const subFieldName = recordFilter.subFieldName;
+  const isSubFieldFilter = isNonEmptyString(subFieldName);
+
+  if (isSubFieldFilter) {
+    switch (subFieldName) {
+      case 'primaryEmail': {
+        return {
+          or: [
+            {
+              [correspondingFieldMetadataItem.name]: {
+                primaryEmail: { eq: '' },
+              } satisfies EmailsFilter,
+            },
+            {
+              [correspondingFieldMetadataItem.name]: {
+                primaryEmail: { is: 'NULL' },
```

Comment:
I'm surprised about this 'NULL' between quote. how do we distinguish it from the 'NULL' string? 
Not tied to this PR but let's confirm that this is the backend behavior